app/views/pages/home.html.erb

```diff
@@ -5,6 +5,9 @@
 
 <div class="cards-container">
   <div class="row">
+    <div id="card-title">
+    <p>Recommended Rides</p>
```

Comment:
all ok, just a thought that we should have clearer styling rules for our final project... all headers should follow same rules (not through css font, weight stuff) but by (bootstrap/our own css variables) and using the right h tags